exporters/committime/collector_base.py

```diff
@@ -264,6 +273,9 @@ def _get_repo_from_build_config(self, build):
         build_config = v1_build_configs.get(
             namespace=build.status.config.namespace, name=build.status.config.name
         )
+        if build.metadata.annotations.buildSpecSourceGitUri:
+            return build.metadata.annotations.buildSpecSourceGitUri
```

Comment:
Is there needed validation of URI at some point? e.g. valid git URI regex?

Reply:
That is handled in the `CommitMetric` class and/or the various collector implementations themselves.